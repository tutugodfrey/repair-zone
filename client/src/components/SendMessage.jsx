import React, { Component } from 'react';
import Div from './elementComponents/Div.jsx';
import Link from './elementComponents/Link.jsx';

export default class Header extends Component {
  sendMessage() {
    return (
      <div>
        send messages
      </div>
    )
  }
  render() {
    return <Div divId="" divClass="" content={this.sendMessage()} />
  }
}