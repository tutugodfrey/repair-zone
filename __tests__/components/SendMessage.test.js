import React from 'react';
import { shallow} from 'enzyme';
import toJson from 'enzyme-to-json';
import SendMessage from '../../client/src/components/SendMessage.jsx';

describe('SendMessage />', () => {
  const wrapper = shallow(
    <SendMessage />
  );
  test('should match the snapshot of the component', () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
