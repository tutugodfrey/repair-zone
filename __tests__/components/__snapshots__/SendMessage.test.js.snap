// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SendMessage /> should match the snapshot of the component 1`] = `
<Div
  content={
    <div>
      Sorry! this feature is not implemented yet
    </div>
  }
  divClass=""
  divId=""
/>
`;
